event-streams/connector.json

```diff
@@ -14,7 +14,7 @@
         "plugin.name": "pgoutput",
         "slot.max.retries": 999999999,
         "database.server.name": "playbook-dispatcher",
-        "table.include.list": "public.runs",
+        "table.include.list": "public.runs,public.run_hosts",
```

Comment:
configuration in `/deploy` will need to be also modified like this

Reply:
I believe this should be done.